src/test/mir-opt/const_allocation3/rustc.main.ConstProp.after.mir

```diff
@@ -31,10 +31,14 @@ alloc2 (static: FOO, size: 8, align: 8) {
 }
 
 alloc8 (size: 140, align: 1) {
-    0x00 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ├──alloc6+0──
-    0x20 │ ──────────┤ 01 ef cd ab 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
-    0x40 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
-    0x60 │ 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+    0x00 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab
+    0x10 │ ab ab ab ab ab ab ab ab ab ab ab ab ├──alloc6+0──
```

Comment:
The part that's too long on the right could use the same "(4 bytes)" annotation that I suggested for pointers not being wide enough in other cases.